WindowsServerDocs/manage/windows-admin-center/azure/azure-extended-network.md

```diff
@@ -0,0 +1,184 @@
+5. Select the Subscription, Resource Group and Virtual Network that you are extending into:
+![Image](../media/azure-extended-network/azure-network.png) 
+The Azure Location and Azure Subnet will be selected automatically.  Click Next: Extended-Network Gateway Setup to proceed.
+6. You will now configure the virtual appliances.  The on-premises gateway should have its information populated automatically:
+![Image](../media/azure-extended-network/azure-network-gateway.png) 
+If it looks correct, you can click Next.
+7. For the Azure virtual appliance you will need to select the Resource Group and VM to use:
+After selecting the VM, you will also need to select the Azure Extended-Network Gateway Subnet CIDR.  Then click Next: Deploy.
+8. Review the summary information then click Deploy to begin the deployment process.  Deployment will take approximately 5-10 minutes.  When deployment is complete you will see the following panel for managing the extended IP addresses, and status should say **OK**:
+![Image](../media/azure-extended-network/installation-complete.png) 
+
+## Manage
+
+Each IP address that you want to be reachable across the extended network will need to be configured.  You can configure up to 250 addresses to extend.
+To extend an address
+
+1. Click on **Add IPv4 Addresses**:
+![Image](../media/azure-extended-network/add-ipv4-addresses.png) 
+
+2. You will see the **Add new IPv4 Addresses** flyout on the right:
+![Image](../media/azure-extended-network/add-ipv4-addresses-panel.png) 
+
+3. Use the **Add** button to manually add an address.  Addresses that you add that are on-premises will be reachable by the Azure Addresses that you add to the Azure Address list, and vice versa.
+
+4. Azure Extended Network scans the network to discovered IP addresses and populates the Suggestion lists based on this scan.  To extend these addresses you must use the dropdown list and select the checkbox next to the discovered address.  Not all addresses will be discovered.  Optionally, use the **Add** button to manually add addresses that are not discovered automatically.
+![Image](../media/azure-extended-network/add-ipv4-addresses-panel-filled.png)
+
+5. Click **Submit** when complete.  You will see the status change to **Updating**, then **Progressing**, and finally back to **OK** when the configuration is complete.
+
+Your addresses are now extended.  Use the Add IPv4 Addresses button to add additional addresses at any time.  If an IP address is no longer in use at either end of the extended network, select the checkbox next to it and select Remove IPv4 Addresses.
+
+If you no longer want to use Azure Extended Network, click the **Remove Azure Extended-Network** button.  This will uninstall the agent from the two virtual appliances and remove the extended IP addresses.  The network will stop being extended.  You will have to re-run the setup after removing if you want to start using the extended network again.
```

Comment:
grammatical/readability comma + referencing pronoun
```suggestion
If you no longer want to use Azure Extended Network, click the **Remove Azure Extended-Network** button.  This will uninstall the agent from the two virtual appliances and remove the extended IP addresses.  The network will stop being extended.  You will have to re-run the setup after removing it, if you want to start using the extended network again.
```